lib/src/widgets/dashboard_screen/start_learning_floating_action_button.dart

```diff
@@ -5,12 +5,12 @@ import 'package:saraka/constants.dart';
 class StartLearningFloatingActionButton extends StatelessWidget {
   @override
   Widget build(BuildContext context) => FloatingActionButton.extended(
-        heroTag: null,
-        backgroundColor: SarakaColors.lightBlue,
+        backgroundColor: SarakaColors.lightRed,
+        foregroundColor: SarakaColors.white,
         icon: Icon(Feather.getIconData('play')),
         label: Text(
           'Start learning',
-          style: SarakaTextStyles.buttonLabel,
+          style: SarakaTextStyles.buttonLabel.copyWith(letterSpacing: 1),
```

Comment:
Why did you change the style to be `letterSpacing: 1` here?

Reply:
as keeping consistency. I followed the "New card" button.

Reply:
Deleted.